### ISSUE TITLE
Is possible serialize models which use torch functions?

### ISSUE THREAD

**Issue by emi-research-dl** [keras-team/keras-core#903]
Would it be possible to serialize loss functions and torch metrics in a keras model?

 I think it would be quite a powerful feature!

Note: I have provided the loss function in the compile method of a subclass of Keras model and have altered the order of logits in compute_loss function:
```
        y_pred = self(x, training=True)  # Forward pass , y_pred is logits
        probs = keras.activations.softmax(y_pred)  # Compute the probabilities for the calc of metrics

        loss = self.compute_loss(y=y_pred, y_pred=y)  # Compute the loss with logits (from_logits=True)

```

Thanks in advanced!!

![image](https://github.com/keras-team/keras-core/assets/140513018/ff88ea60-fd41-44eb-9c84-1c6571bc075a)

**Comment by fchollet** [comment]
Not yet, but we want to add this feature. Here's a related issue: https://github.com/keras-team/keras-core/issues/901

**Comment by emi-research-dl** [comment]
It would be great!!! Thanks @fchollet :)

**Comment by emi-research-dl** [comment]
Hi everyone!!! Here is a colab for reproduce the issue: https://colab.research.google.com/drive/1kFQ4_2qPlHm4P63Qb2ry-lC0OQ_zY0RJ?usp=sharing

**Comment by emi-research-dl** [comment]
@fchollet I thought maybe you could make a class wrapper that wraps the base class of torch.nn to implement the get_config and from_config methods, so that you could serialize. I don't have much knowledge of the internal implementation of Keras and I don't know if you could...

Thanks in advanced guys!

**Comment by qlzh727** [comment]
Closing this as a duplication against https://github.com/keras-team/keras-core/issues/901. we will track the fix there.
